theory.md

```diff
@@ -0,0 +1,45 @@
+# Task 2: Theory Questions
+
+## Question 1
+**Fill in the blank:**
+**Git is an example of a_______________system.**
+Git is an example of a__Distributed Version Control_____________system.
+
+## Question 2
+**Name a code hosting service that uses Git.**
+
+GitLab
+
+## Question 3
+**There are 3 states in Git that files can be in. What are these states?**
+
+Modified: The file has been changed but has not been committed to the repository.
+Staged: The file has been modified and is ready to be committed to the repository.
+Committed: The data is safely stored in your local database.
+
+## Question 4
+**PR and repo are common abbreviations in the world of Git. What do they mean?**
+
+PR is the abbreviation for Pull Request. It allows you to propose changes to code on a Git repository. Creating a pull request means you are requesting that the owner of the repository pull your changes into the main branch of their repository.
+
+Repo is the abbreviation for repository. In Git, a repository is used to store a collection of files and any changes made to them.
+
+## Question 5
+**What is the git command that updates the local repository with changes from the remote repository?**
+
+The git pull comand.
+
+## Question 6
+**What is the git command that gets the changes from the remote repository but doesn’t update your local repository?**
+
+The git fetch command.
+
+## Question 7
+**What is the git command that updates the remote repository with local repository code?**
+
+The git push command.
+
+## Question 8
+**What does the git commit command need to work?**
+
+Files will need to be added to the staging area.
```

Comment:
Question 8 - Incorrect answer, Correct answer is `git commit -m 'commit message'`
Marks (12/14)